Acceptance tests fail locally when running against a test buildkite org

I've tried to run acceptance tests locally (https://github.com/buildkite/terraform-provider-buildkite/commit/10576bdb6b6b9dcb987300a7868ee9cf7a3fe7f6). I created a new buildkite org (and enabled "Teams" for that org), generated a new API Token for my user and run the tests. For some reason a single test is failing `TestAccPipelineSchedule_import`. Am I doing something wrong?

```
$ BUILDKITE_ORGANIZATION=********* BUILDKITE_API_TOKEN=********** make testacc
TF_ACC=1 go test -v ./...
?       github.com/buildkite/terraform-provider-buildkite       [no test files]
=== RUN   TestAccDataPipeline_read
--- PASS: TestAccDataPipeline_read (23.05s)
=== RUN   TestProvider
--- PASS: TestProvider (0.00s)
=== RUN   TestProvider_impl
--- PASS: TestProvider_impl (0.00s)
=== RUN   TestAccAgentToken_add_remove
--- PASS: TestAccAgentToken_add_remove (27.26s)
=== RUN   TestAccAgentToken_update
--- PASS: TestAccAgentToken_update (32.67s)
=== RUN   TestAccAgentToken_import
--- PASS: TestAccAgentToken_import (28.70s)
=== RUN   TestAccPipelineSchedule_add_remove
--- PASS: TestAccPipelineSchedule_add_remove (25.47s)
=== RUN   TestAccPipelineSchedule_add_remove_withteams
--- PASS: TestAccPipelineSchedule_add_remove_withteams (26.87s)
=== RUN   TestAccPipelineSchedule_update
--- PASS: TestAccPipelineSchedule_update (38.38s)
=== RUN   TestAccPipelineSchedule_update_withteams
--- PASS: TestAccPipelineSchedule_update_withteams (36.58s)
=== RUN   TestAccPipelineSchedule_import
    TestAccPipelineSchedule_import: resource_pipeline_schedule_test.go:152: Step 2/2 error running import: 2021/03/10 09:43:37 [DEBUG] Using modified User-Agent: Terraform/0.12.29 HashiCorp-terraform-exec/0.10.0
        
        Error: nil entry in ImportState results. This is always a bug with
        the resource that is being imported. Please report this as
        a bug to Terraform.
        
        
--- FAIL: TestAccPipelineSchedule_import (25.52s)
=== RUN   TestAccPipelineSchedule_disappears
--- PASS: TestAccPipelineSchedule_disappears (22.35s)
=== RUN   TestAccPipeline_add_remove
--- PASS: TestAccPipeline_add_remove (20.83s)
=== RUN   TestAccPipeline_add_remove_complex
--- PASS: TestAccPipeline_add_remove_complex (20.20s)
=== RUN   TestAccPipeline_add_remove_withteams
--- PASS: TestAccPipeline_add_remove_withteams (20.24s)
=== RUN   TestAccPipeline_update
--- PASS: TestAccPipeline_update (34.10s)
=== RUN   TestAccPipeline_update_withteams
--- PASS: TestAccPipeline_update_withteams (35.70s)
=== RUN   TestAccPipeline_import
--- PASS: TestAccPipeline_import (25.85s)
=== RUN   TestAccPipeline_disappears
--- PASS: TestAccPipeline_disappears (17.91s)
=== RUN   TestAccTeam_add_remove
--- PASS: TestAccTeam_add_remove (18.84s)
=== RUN   TestAccTeam_update
--- PASS: TestAccTeam_update (50.31s)
=== RUN   TestAccTeam_import
--- PASS: TestAccTeam_import (27.55s)
=== RUN   TestAccTeam_disappears
--- PASS: TestAccTeam_disappears (19.28s)
FAIL
FAIL    github.com/buildkite/terraform-provider-buildkite/buildkite     578.133s
FAIL
make: *** [testacc] Error 1
```

--- a/buildkite/resource_pipeline_schedule_test.go
+++ b/buildkite/resource_pipeline_schedule_test.go
@@ -3,6 +3,7 @@ package buildkite
 import (
 	"context"
 	"fmt"
+	"os"
 	"testing"
 
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
@@ -257,7 +258,7 @@ func testAccCheckPipelineScheduleRemoteValues(resourcePipeline *PipelineNode, re
 
 func testAccGetImportPipelineScheduleSlug(resourceSchedule *PipelineScheduleNode) resource.ImportStateIdFunc {
 	return func(s *terraform.State) (string, error) {
-		slug := fmt.Sprintf("%s/%s/%s", "buildkite-terraform-provider-test-org", "test-pipeline-foo", resourceSchedule.UUID)
+		slug := fmt.Sprintf("%s/%s/%s", os.Getenv("BUILDKITE_ORGANIZATION"), "test-pipeline-foo", resourceSchedule.UUID)
 		return slug, nil
 	}
 }

